### ISSUE TITLE
[Question] Its possible to serve/test while training?

### ISSUE THREAD

**Issue by kauegimenes** [suriyadeepan/easy_seq2seq#1]
Its possible to serve/test while training?
Thanks.

**Comment by suriyadeepan** [comment]
Yes, it is. But you will be serving the old model, not the one that is being updated by the train function. 

### Train

`python execute.py # with mode = train, in seq2seq.ini`

### Test

`python ui/app.py # with mode = serve, in seq2seq_serve.ini`

What you can do is, you can periodically check for updates in the model, and reinitialize your test/serve session with new model parameters. I will try to implement this in the future. Thanks for suggesting it.

**Comment by kauegimenes** [comment]
@suriyadeepan Thanks for the reply!

I tried setting mode = serve and using the command `python3 ui/app.py`
But i get this error:
`ImportError: No module named 'execute'
`

**Comment by kauegimenes** [comment]
@suriyadeepan Fixed. 
Auto update the serve to the last model would be a nice feature!

**Comment by harikrishnanvh** [comment]
@kauegimenes how did you fix the error 'ImportError: No module named 'execute' . Can you please help

**Comment by jrthom18** [comment]
@harikrishnanvh add the following imports: above the lines in ui/app.py `import tensorflow as tf
import execute` add the lines: `import sys
import os.static_url_path
sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), os.path.pardir)))`

**Comment by harikrishnanvh** [comment]
@jrthom18 Thanks a ton. It worked beautifully.

**Comment by PW486** [comment]
@jrthom18 
import os.static_url_path
ImportError: No module named static_url_path
I have made an error here

import os.path
changed it this way. works well. THANKYOU!

**Comment by kumar-shridhar** [comment]
Hi,
I am getting this error:

File "ui/app.py", line 33, in <module>
    sess, model, enc_vocab, rev_dec_vocab = execute.init_session(sess, conf='seq2seq_serve.ini')
AttributeError: 'module' object has no attribute 'init_session'

Any help will be appreciated.
